llvm/lib/Transforms/Instrumentation/AddressSanitizer.cpp

```diff
@@ -2449,6 +2451,77 @@ Instruction *ModuleAddressSanitizer::CreateAsanModuleDtor(Module &M) {
   return ReturnInst::Create(*C, AsanDtorBB);
 }
 
+void ModuleAddressSanitizer::instrumentDeviceGlobal(IRBuilder<> &IRB,
```

Comment:
add lit for change in this pass?

Reply:
I originally thought on this. But looks like opt tool doesn't support spir64 target triple. I don't know if there are other ways to test it.

Reply:
you can add lit tests like https://github.com/intel/llvm/pull/13503

Reply:
Thanks, done.